app/templates/search/_results_wrapper.html

```diff
@@ -0,0 +1,8 @@
+<div id="js-dm-live-search-results" class="js-dm-live-search-fade">
+  {% if framework_family == 'g-cloud' %}
+    {% include 'search/_services_results.html' %}
```

Comment:
I wonder if there's something we could do here in terms of creating a directory for each framework family, and building the string for the `include` from that. Arguably not as clear as this, not sure, hmm :)